consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/validation/RoundChangeMessageValidator.java

```diff
@@ -115,13 +115,13 @@ private boolean validatePreprepareCertificateRound(
       final ConsensusRoundIdentifier roundChangeTarget) {
 
     if (prepareCertRound.getSequenceNumber() != roundChangeTarget.getSequenceNumber()) {
-      LOG.info("Invalid RoundChange message, PreprepareCertificate is not for local chain height.");
+      LOG.info("Invalid RoundChange message, PreparedCertificate is not for local chain height.");
```

Comment:
should this be ProposalCertificate rather than PreparedCertificate?

Reply:
no - it's still a PreparedCertificate I do believe (as in, the node is in the prepared state).